lib/carbon/cache.py

```diff
@@ -42,6 +45,10 @@ def gen_queue(self):
 
   def store(self, metric, datapoint):
     self.size += 1
+    dbFilePath = getFilesystemPath(metric)
+    if settings.CACHE_CREATE_WHISPER_FILES:
+      if (len(self[metric]) == 0):
+        queueWhisperCreate(metric, dbFilePath)
     self[metric].append(datapoint)
```

Comment:
It does not appear that this branch includes that exception handling code in the new `createWhisperFile` method.